src/annotations/AnnotationParser.php

```diff
@@ -13,6 +13,10 @@
 
 namespace mindplay\annotations;
 
+if (!defined('T_TRAIT')) {
+    define('mindplay\\annotations\\T_TRAIT', -2);
```

Comment:
Please replace the `'mindplay\\annotations\\T_TRAIT'` with `__NAMESPACE__ . '\\T_TRAIT'`. That should help reduce duplication of namespace as strings in code.